lib/deployment-tasks.js

```diff
@@ -88,6 +88,12 @@ function buildImages(registry) {
 	return function(buildOptions) {
 		logger.info('Building Docker Images');
 		logger.info('Will build the following images:');
+
+		for(let name in buildOptions.images)
```

Comment:
_.filter or any type of filter destroys object keys, why I ended up using for..in